### PR TITLE
Close swiped song item

### PR DESCRIPTION
**Pull request by pavelLyolya**

### CODE REVIEW

File: src/screens/Favorites/Favorites.js [hunk 1]
```diff
@@ -92,22 +128,34 @@ class Favorites extends Component {
...
     }
     return (
-      <View style={styles.container}>
-        <FlatList
-          contentContainerStyle={styles.flaList}
-          data={this.props.favorites.sort((a, b) => a.number - b.number)}
-          keyExtractor={this.keyExtractor}
-          renderItem={this.renderItem}
+      <>
+        <ScrollView
```
Comment (nataly-chobat) [inline src/screens/Favorites/Favorites.js]: FlatList includes scroll view, Why did you add Scroll View?

Comment (pavelLyolya) [reply src/screens/Favorites/Favorites.js]: I didn't know how to use .scrollTo with FlatList... :)
